public/locales/en.json

```diff
@@ -455,6 +455,15 @@
         "copyError": "Error while copying!",
         "errorOccurred": "An error occurred"
     },
+    "global": {
+        "modal": {
+            "reloadApp": {
+                "title": "Update available",
+                "content": "An update for hedgedoc is available. Please reload every hedgedoc tab.",
+                "reloadButton": "Reload app"
```

Comment:
This in total is quite similar to another message we already have: `editor.error.newVersion`. To reduce the load on the translators, maybe we could use that instead or merge them?

Reply:
I would remove it becaus its no error and we have a new modal for that.

Reply:
Should we move the already translated strings to the new message if applicable?

Reply:
It's another translation. And tbh I don't like the old translation.